tests/python/unittest/test_sparse_operator.py

```diff
@@ -1675,6 +1675,30 @@ def check_sparse_embedding(in_dim, out_dim, batch, densities, deterministic):
     check_sparse_embedding(in_dim, out_dim, batch, densities, False)
 
 
+@with_seed()
+def test_sparse_broadcast_mul_div():
+    from scipy.sparse import random, csr_matrix
```

Comment:
I don't think `scipy.sparse.csr_matrix` is used here

Reply:
Removed